### ISSUE TITLE
Update to REX-Ray 0.6.x

### ISSUE THREAD

**Issue by h0tbird** [katosys/kato#50]
This task needs https://github.com/codedellemc/libstorage/pull/325 to be released. Find below the changes needed:
```
diff --git a/udata/fragments.go b/udata/fragments.go
index aa033a0..a62b0cf 100644
--- a/udata/fragments.go
+++ b/udata/fragments.go
@@ -104,8 +104,14 @@ write_files:`,
 {{- if .RexrayStorageDriver }}
    content: |
     rexray:
-     storageDrivers:
-     - {{.RexrayStorageDriver}}
+     logLevel: warn
+    libstorage:
+     embedded: true
+     service: {{.RexrayStorageDriver}}
+     server:
+      services:
+       virtualbox:
+        driver: virtualbox
     virtualbox:
      endpoint: http://` + d.RexrayEndpointIP + `:18083
      volumePath: ` + os.Getenv("HOME") + `/VirtualBox Volumes
@@ -1025,14 +1031,14 @@ coreos:
      Restart=always
      RestartSec=10
      TimeoutStartSec=0
+     KillMode=process
      EnvironmentFile=/etc/rexray/rexray.env
-     ExecStartPre=-/bin/bash -c '\
-       REXRAY_URL=https://emccode.bintray.com/rexray/stable/0.3.3/rexray-Linux-i386-0.3.3.tar.gz; \
-       [ -f /opt/bin/rexray ] || { curl -sL $${REXRAY_URL} | tar -xz -C /opt/bin; }; \
-       [ -x /opt/bin/rexray ] || { chmod +x /opt/bin/rexray; }'
+     Environment=URL=https://dl.bintray.com/emccode/rexray/stable/0.6.0/rexray-Linux-x86_64-0.6.0.tar.gz
+     ExecStartPre=-/bin/bash -c " \
+       [ -f /opt/bin/rexray ] || { curl -sL ${URL} | tar -xz -C /opt/bin; }; \
+       [ -x /opt/bin/rexray ] || { chmod +x /opt/bin/rexray; }"
      ExecStart=/opt/bin/rexray start -f
      ExecReload=/bin/kill -HUP $MAINPID
-     KillMode=process

      [Install]
      WantedBy=kato.target`,
```

**Comment by h0tbird** [comment]
Will be released in *libstorage* `v0.3.3`

**Comment by h0tbird** [comment]
Done
